ensime_shared/config.py

```diff
@@ -4,17 +4,20 @@
 }
 
 feedback = {
-    "start_message": "Ensime server has been started...",
-    "unknown_symbol": "Symbol not found",
-    "indexer_ready": "Ensime indexer is ready",
     "analyzer_ready": "Ensime analyzer is ready",
-    "module_missing": "{} missing: do a `pip install {}` and restart vim",
-    "warn_classpath": "Execute :EnClasspath to set a classpath",
-    "missing_debug_class": "You must specify a class to debug",
-    "notify_break": "Execution breaked at {} {}",
     "displayed_type": "The type {} has been displayed",
     "failed_refactoring":
         "The refactoring could not be applied (more info at logs)",
-    "unhandled_response": "Response {} has not been handled",
+    "full_types_enabled_on": "[ensime] Qualified type display enabled",
+    "full_types_enabled_off": "[ensime] Qualified type display disabled",
+    "indexer_ready": "Ensime indexer is ready",
+    "manual_doc": "Go to {}",
+    "missing_debug_class": "You must specify a class to debug",
+    "module_missing": "{} missing: do a `pip install {}` and restart vim",
+    "notify_break": "Execution breaked at {} {}",
     "spawned_browser": "Opened tab {}",
-    "manual_doc": "Go to {}"}
+    "start_message": "Ensime server has been started...",
+    "unhandled_response": "Response {} has not been handled",
+    "unknown_symbol": "Symbol not found",
+    "warn_classpath": "Execute :EnClasspath to set a classpath"
+}
```

Comment:
Sorry about the noise here, all that's changed is adding the `full_types_enabled_*` messages but I went ahead and sorted these lines before it keeps growing…